chrome/Echofon/content/overlay.js

```diff
@@ -868,8 +868,8 @@ var EchofonOverlay = {
 
   onFinishOAuth: function(user_id) {
     var account = EchofonAccountManager.instance().get(user_id);
-    if (EchofonCommon.pref().getIntPref("activeUserId") == 0) {
-      EchofonCommon.pref().setIntPref("activeUserId", account.user_id);
+    if (EchofonCommon.pref().getCharPref("activeUserIdStr") == 0) {
```

Comment:
I'm thinking of just writing `if (EchofonCommon.pref().getCharPref("activeUserIdStr")) {`. Would be easier right ?